homeassistant/components/light/hue.py

```diff
@@ -95,16 +103,19 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
             socket.gethostbyname(host) in _CONFIGURED_BRIDGES:
         return
 
-    setup_bridge(host, hass, add_devices, filename, allow_unreachable)
+    setup_bridge(host, username, hass, add_devices, filename,
+                 allow_unreachable)
 
 
-def setup_bridge(host, hass, add_devices, filename, allow_unreachable):
+def setup_bridge(host, username, hass, add_devices, filename,
+                 allow_unreachable):
     """Setup a phue bridge based on host parameter."""
     import phue
 
     try:
         bridge = phue.Bridge(
             host,
+            username=username,
             config_file_path=hass.config.path(filename))
     except ConnectionRefusedError:  # Wrong host was given
         _LOGGER.error("Error connecting to the Hue bridge at %s", host)
```

Comment:
Could be beneficial to add username in here? So it can give a more fruitful message for the user to fix the problem. 👍